test/FunctionalTests/Microsoft.OData.Edm.Tests/ScenarioTests/OasisRelationshipChangesAcceptanceTests.cs

```diff
@@ -463,6 +463,67 @@ public void ValidationShouldFailIfNavigationParterIsSpecifiedButCannotBeFound()
                 });
         }
 
+        [Fact]
+        public void ValidationShouldFailIfEnumMemberIsSpecifiedButCannotBeFound()
+        {
+            IEdmModel model = GetEdmModel(@"<EnumMember>TestNS2.UnknowColor/Blue</EnumMember>");
+            IEnumerable<EdmError> errors;
+            model.Validate(out errors).Should().BeFalse();
+            errors.Should().HaveCount(1);
+            errors.Should().Contain(e => e.ErrorCode == EdmErrorCode.BadUnresolvedEnumMember && e.ErrorMessage == ErrorStrings.Bad_UnresolvedEnumMember("Blue"));
+        }
+
+        [Fact]
+        public void ValidationShouldFailIfEnumMemberIsSpecifiedButCannotBeFoundTheMember()
+        {
+            IEdmModel model = GetEdmModel(@"<EnumMember>TestNS2.Color/Yellow</EnumMember>");

```

Comment:
Maybe update the color from Yellow to "UnknownMember" to be consistent with the previous test case?

Reply:
#resolved.